lightning_uq_box/datamodules/toy_heteroscedastic.py

```diff
@@ -94,6 +94,9 @@ def __init__(
                 signature f(x, noise: bool)
             noise_seed: random seed for x points positions and y noise
             split_seed: random seed for train/test/val split
+
+        .. versionchanged:: 0.2.0
```

Comment:
Usually these are only the first two digits (0.2), the API should never change during a patch release.